### ISSUE TITLE
Benchmarking h2o, h2o not compling on MacOS

### ISSUE THREAD

**Issue by enilsen16** [potatosalad/phoenix_h2load#2]
Hey, I have been messing around a bit with h2o and I stumbled on this repository a few days ago. I was able to get most of the makefile tests working with docker. The h2o commands don't seem to work though when I try to build your fork of h2o on both docker and MacOS.

Are there other environment variables or dependencies needed to build your erlang-h2o fork?

**Comment by potatosalad** [comment]
Yes, it's really clunky right now, but you'll need to set the environment variable `H2O_VERSION=2` for h2o to be enabled. Currently, I've been able to build it on macOS, Linux (Arch), and SmartOS. I haven't been able to build it using the docker files in this project yet due to issues compiling LibreSSL on Alpine Linux (specifically that it can't find `linux/sysctl.h` or something like that).

Building on macOS is probably your best route to success right now.  However, if there are any compilation errors, feel free to post them here and I'll see if I can help.
